src/particles/elements/Drift.H

```diff
@@ -155,6 +156,41 @@ namespace impactx
             refpart.s = s + slice_ds;
 
         }
+
+        /** This function returns the linear transport map.
+         *  
+         * @returns 6x6 transport matrix
+         */
+        AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
+
+        amrex::Array2D<amrex::ParticleReal, 1, 6, 1, 6>
+        transport_map(RefPart & AMREX_RESTRICT refpart) {
+            
+            using namespace amrex::literals; // for _rt and _prt
+
+            amrex::Array2D<amrex::ParticleReal, 1, 6, 1, 6> R;
```

Comment:
I think this is not zero-intialized.
```suggestion
            LinearTransport::Map6x6 R;
            for (int i=1; i<=6; ++i) {
                for (int j=1; j<=6; ++j) {
                    amrex::ParticleReal R_ij = (i == j) ? 1.0 : 0.0;
                    transport_map(i, j) = R_ij;
                }
            }
```

We can make this a unity matrix and the zero-matrix also a helper function of  `elements::LinearTransport` so we can start in actual elements from there instead of copying this double loop around.

Reply:
This requires `elements::LinearTransport::Map6x6 = {};`